### PR TITLE
add channels.online to Channels

### PR DESCRIPTION
**Pull request by chrisstime**
Lists all online users within a specified channel

### CODE REVIEW

File: lib/rocket_chat/messages/channel.rb [hunk 1]
```diff
@@ -40,6 +40,21 @@ def list(offset: nil, count: nil, sort: nil, fields: nil, query: nil)
         response['channels'].map { |hash| RocketChat::Room.new hash } if response['success']
       end
 
+      #
+      # channels.online REST API
+      # @param [String] room_id Rocket.Chat room id
+      # @return [Users]
+      # @raise [HTTPError, StatusError]
+      #
+      def online(room_id)
```
Comment (abrom) [inline lib/rocket_chat/messages/channel.rb]: Could you update this to use the same method signature as the `join` method?

ie `def online(room_id: nil, name: nil)` then pass the param to the body using `body: room_params(room_id, name)` helper. 

You'll need to update your params above and the README docs

Comment (chrisstime) [reply lib/rocket_chat/messages/channel.rb]: I'm not sure why I keep getting `NameError:
       undefined local variable or method `online' for #<RSpec::ExampleGroups::RocketChatMessagesChannel::Online::ValidSession:0x00007f8fff0ea0d8>` when stubbing the call in the specs... (I've likely missed something) so let me know

File: lib/rocket_chat/messages/channel.rb [hunk 2]
```diff
@@ -40,6 +41,21 @@ def list(offset: nil, count: nil, sort: nil, fields: nil, query: nil)
...
 
+      #
+      # channels.online REST API
+      # @param [String] room_id Rocket.Chat room id
+      # @return [Users]
+      # @raise [HTTPError, StatusError]
+      #
+      def online(room_id: nil, name: nil)
+        response = session.request_json(
+          'api/v1/channels.online',
```
Comment (abrom) [inline lib/rocket_chat/messages/channel.rb]: where is the leading `/` ?

File: spec/messages/channel_spec.rb [hunk 3]
```diff
@@ -52,4 +52,78 @@
...
+              _id: 'rocketID2',
+              username: 'rocketUserName2'
+            }
+          ]
+        }.to_json,
+        status: 200
+      }
+    end
+
+    let(:invalid_room_body) do
```
Comment (abrom) [inline spec/messages/channel_spec.rb]: why does the `invalid_room_body` return success ??

File: spec/messages/channel_spec.rb [hunk 4]
```diff
@@ -52,4 +52,78 @@
       end
     end
   end
+
+  describe '#online' do
+    let(:online_users_body) do
```
Comment (abrom) [inline spec/messages/channel_spec.rb]: also, this seems more like it'd be better named `online_users_response` given that it includes both the body and status (same below)

**Pull request merged**